straxen/analyses/waveform_plot.py

```diff
@@ -145,19 +147,33 @@ def plot_records_matrix(context, run_id,
                     max_samples=max_samples,
                     ignore_max_sample_warning=ignore_max_sample_warning,
                     **kwargs)
+    if group_by:
```

Comment:
Maybe an explicit `if group_by is not None:`? I've had linters complain at me about this

Reply:
Thanks good idea